### PR TITLE
Add new timer implementation api doc

### PR DESCRIPTION
**Pull request by Glukoosi**
Move timer post, patch and delete to seperate endpoints

### CODE REVIEW

File: api-doc/openapi.yaml [hunk 1]
```diff
@@ -192,10 +192,52 @@ paths:
...
+  /timers/{timerId}:
+    parameters:
+      - name: timerId
+        in: path
+        required: true
+        description: Timer id (uuid).
+        schema:
+          type: string
+          format: uuid
+    post:
```
Comment (Moetto) [inline api-doc/openapi.yaml:208]: Post should be under `/timers`

File: api-doc/openapi.yaml [hunk 2]
```diff
@@ -1014,6 +1110,70 @@ components:
...
+          type: string
+        start_time:
+          type: string
+          format: date-time
+        end_time:
+          type: string
+          format: date-time
+      required:
+        - id
+        - timestamps
```
Comment (Moetto) [inline api-doc/openapi.yaml:1129]:
```suggestion
        - start_time
        - end_time
```
I made name mandatory as well, but can drop that requirement

File: api-doc/openapi.yaml [hunk 3]
```diff
@@ -1014,6 +1110,70 @@ components:
...
+          type: string
+          format: date-time
+          nullable: true
+        end_time:
+          type: string
+          format: date-time
+          nullable: true
+      required:
+        - start_time
+        - end_time
```
Comment (Moetto) [inline api-doc/openapi.yaml:1150]: Not required I think? Or did you have something specific in mind with nullable required field?

```suggestion
```

File: api-doc/openapi.yaml [hunk 4]
```diff
@@ -1014,6 +1110,70 @@ components:
...
+        start_time:
+          type: string
+          format: date-time
+          nullable: true
+        end_time:
+          type: string
+          format: date-time
+          nullable: true
+      required:
+        - id
```
Comment (Moetto) [inline api-doc/openapi.yaml:1172]: Not required in patch I think
```suggestion
```

File: api-doc/openapi.yaml (hunk 2 above)
Comment (Glukoosi) [reply api-doc/openapi.yaml]: name is now mandatory

File: api-doc/openapi.yaml (hunk 3 above)
Comment (Glukoosi) [reply api-doc/openapi.yaml]: Not required

**Review by Moetto — APPROVED** [review]

**Pull request merged**